core/utils.go

```diff
@@ -75,6 +75,13 @@ func isPrimary(view uint64, id uint32, n uint32) bool {
 	return uint64(id) == view%uint64(n)
 }
 
+func getTrimmedMsg(msg string) string {
+	if len(msg) > logMaxPayloadWidth {
```

Comment:
There is a bit of dissonance between the function name and the name of this constant. The constant name relates to the payload, whereas the function name doesn't seem to reflect that. Maybe rename the constant more generically, or pass its value as a parameter.